### PR TITLE
Update the README and fix building with no GPUs

### PR DESCRIPTION
**Pull request by krzysz00**

### CODE REVIEW

**Comment by nirvedhmeshram** [comment]
Thanks for this. I was looking for build instructions for the repository.

File: README.md [hunk 1]
```diff
@@ -1,113 +1,69 @@
...
+This project also includes cod that translates from TOSA to kernels, see
+`mlir/test/fusion` for examples of how to invoke it.
+
+In general (with all invocations given from the build directory)
+- `./bin/miopen-gen` generates high-level convolution operations and
+  host code. Many of the options control data layout, size, etc, but some other
+  useful flags are:
+    - `-x2` (which enables mfma usage)
+    - `-ph` (which causes host code to be generated)
+    - `-pv` (which makes the host code validtae the results against a reference)
```
Comment (zhanglx13) [inline README.md:52]: Typo "validtae"

File: README.md [hunk 2]
```diff
@@ -1,113 +1,69 @@
...
+    - `-pv_with_gpu` (which uses a GPU validator instead)
+    - `-pr` (which prints kkrnel results)
```
Comment (zhanglx13) [inline README.md:54]: Typo "kkrnel"

**Comment by krzysz00** [comment]
@nirvedhmeshram The fix for y'all's build issue should be on this branch

**Review by jerryyin — APPROVED** [review]

**Pull request merged**